ej/models.py

```diff
@@ -8,6 +8,8 @@
 class Problem(db.Entity):
     title = Required(str, unique=True)
     time_limit = Required(int)
+    custom_check = Required(bool)
```

Comment:
I was thinking about a future form to register a new problem where it would have a checkbox `custom check`, thats why.